builtin/game/chatcommands.lua

```diff
@@ -679,19 +679,36 @@ core.register_chatcommand("status", {
 })
 
 core.register_chatcommand("time", {
-	params = "<0...24000>",
+	params = "<0..23>:<0..59> | <0..24000>",
 	description = "set time of day",
 	privs = {settime=true},
 	func = function(name, param)
 		if param == "" then
-			return false, "Missing time."
+			local current_time = math.floor(core.get_timeofday() * 1440)
+			return true, ("Current time is %2d:%2d"):format(
+					(current_time - (current_time % 60)) / 60, (current_time % 60))
+		end
+		local hour, minutes = param:match("^(%d+):(%d+)$")
+		if hour == nil or minutes == nil then
+			local newtime = tonumber(param)
+			if newtime ~= nil then
+				-- Backward compatibility.
```

Comment:
Quote from @PilzAdam :

> You should make this backwards compatible.

I interpreted this meaning that some people would be used to the old `0..24000` system.